packages/tasit-action/src/erc721/ERC721Full.test.js

```diff
@@ -154,7 +154,7 @@ describe("TasitAction.ERC721.ERC721Full", () => {
       expect(event.to).to.equal(bob.address);
       expect(event.tokenId.toNumber()).to.equal(tokenId);
 
-      await confirmBalances(erc721, [ana.address, bob.address], [0, 1]);
+      await tokenBalancesAreEqual(erc721, [ana.address, bob.address], [0, 1]);
```

Comment:
I won't have this gate merging, but the function name as it's worded right now kind of makes it sound like Ana's and Bob's addresses should be equal

Reply:
Maybe `expectExactBalances` and `expectMinimumBalances` would work instead?

Reply:
`expect` as opposed to `confirm` since `confirm` is a loaded word in the Ethereum ecosystem

Reply:
I liked `expectExactBalances` and `expectMinimumBalances`, I'll rename these.

Reply:
Great, thanks

Reply:
Resolving this since it changed in https://github.com/tasitlabs/TasitSDK/pull/281